open-api/rest-catalog-open-api.yaml

```
@@ -2106,6 +2213,32 @@ components:
           items:
             $ref: '#/components/schemas/PartitionStatisticsFile'
 
+    PlanTask:
+      description:
+        An opaque JSON object that contains information provided by the REST server,
+        to be utilized by clients for distributed table scan planning, should be supplied
```

```suggestion
        to be utilized by clients for distributed table scan planning; should be supplied
```
having a , there doesn't feel right

---

will fix